hub/scripts.js

```diff
@@ -245,6 +245,26 @@
     });
   }
 
+  /**
+   * Loads a JS file on the page
+   * @param {String} src The path to the JS file
+   * @param {Object} param The parameters used to load the file
+   */
+  function loadJS(src, param) {
+    if (isNonEmptyString(src)) {
```

Comment:
First time I look at the code but why do you need a function (with a complicated name ;) ) while you can simply do: `str !== ''` ?

Reply:
we want to make sure a string is passed and it's not empty; using `str !== ''` will evaluate as true even when `str` is actually an Array or Object.